email

```diff
@@ -238,15 +238,28 @@ fi
 # without this mailx returns without error even if sending has failed
 export sendwait=1
 _mailx_version=$($Mailx -V)
-if echo $_mailx_version|grep -q "GNU Mailutils"; then
-    echo "DEBUG:echo '$MailBody' | $Mailx -n -r '$MailFrom' -s '$MailSubject' -a $MAILPDF $MailTo" 1>&2
-    _mailx_status=$(echo "$MailBody" | $Mailx -n -r "$MailFrom" -s "$MailSubject" -a $MAILPDF $MailTo 2>&1)
-    _mailx_rc=$?
-else
-    echo "DEBUG:echo '$MailBody' | $Mailx -n -r '$MailFrom' -s '$MailSubject' -A $MAILPDF $MailTo" 1>&2
-    _mailx_status=$(echo "$MailBody" | $Mailx -n -r "$MailFrom" -s "$MailSubject" -A $MAILPDF $MailTo 2>&1)
-    _mailx_rc=$?
-fi
+# specific option(s) to use
+case $_mailx_version in
+    *"mailx-wrapper"*)
+	export -n MAILRC
+	mailrc_opt="--smtp $MAILRC"
+        attach_opt=-A
+	;;
+    *"GNU Mailutils"*)
+        attach_opt=-A
+        ;;
+    *)
+        attach_opt=-a
+        ;;
+esac
+
+echo "DEBUG:echo '$MailBody' \
+    | $Mailx $mailrc_opt -n -r '$MailFrom' -s '$MailSubject' '|' Job:'$1' $attach_opt '$MAILPDF' $MailTo" 1>&2
+_mailx_status=$(
+    echo "$MailBody" \
+        | $Mailx $mailrc_opt -n -r "$MailFrom" -s "$MailSubject | Job:$1" $attach_opt "$MAILPDF" "$MailTo" 2>&1
```

Comment:
This mostly looks OK, but default output format should not change. So appending the job information into the subject as default is not OK. It makes sense to have that, though - but there might be other useful info in the subject as well, which can get messy quickly. So my suggestion is to drop that from this pull request, and create a new issue about adding extra information into the subject line.